src/Parsing/Impl/Resources.resx

```diff
@@ -258,6 +258,18 @@
   <data name="NamedExpressionCtxtErrorMsg" xml:space="preserve">
     <value>Named expression must be parenthesized in this context</value>
   </data>
+  <data name="NamedExpressionInClassBodyErrorMsg" xml:space="preserve">
+    <value>assignment expression within a comprehension cannot be used in a class body</value>
```

Comment:
First letter capitalization?

Reply:
These are all the same as cpython, which doesn't capitalize at the start. The named ones that exist are the ones that are inconsistent (which I can fix).